code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Food-Reagents.dm

```diff
@@ -499,4 +502,9 @@
 /datum/reagent/fat/concentrated/reaction_turf(turf/simulated/T, reac_volume)
 	if (!istype(T)) return
 	if(reac_volume >= 1)
+		T.MakeSlippery(1)
+
+/datum/reagent/cfat/concentrated/reaction_turf(turf/simulated/T, reac_volume)
```

Comment:
>fat and cfat typepaths

If cfat isn't a child of fat, I should have fixed that myself. Thank you for bringing that up. I'll confirm what's correct later.